sdk/core/Azure.Core/tests/AzureSasCredentialSynchronousPolicyTests.cs

```diff
@@ -73,5 +73,24 @@ public async Task VerifyRetryNonEmptyQuery(string signatureValue)
 
             Assert.AreEqual("?foo=bar&sig=test_signature_value", transport.Requests[0].Uri.Query);
         }
+
+        [TestCase("sig=test_signature_value")]
+        [TestCase("?sig=test_signature_value")]
+        public async Task IgnoreDuplicateSas(string signatureValue)
```

Comment:
Did you make a change to the policy and/or credential or was this just an additional test to verify the existing behavior?

Reply:
Just to validate the existing behavior